frontend/apps/reader/modules/readerfont.lua

```diff
@@ -231,4 +258,26 @@ function ReaderFont:addToMainMenu(menu_items)
     }
 end
 
+function ReaderFont:onPinch()
+    local info = Notification:new{text = _("Changing font size")}
+    UIManager:show(info)
+    UIManager:forceRePaint()
+    UIManager:nextTick(function()
+        self:onChangeSize("decrease")
+        UIManager:close(info)
+    end)
+    return true
+end
+
+function ReaderFont:onSpread()
+    local info = Notification:new{text = _("Changing font size")}
+    UIManager:show(info)
+    UIManager:forceRePaint()
```

Comment:
```lua
    local info = Notification:new{text = _("Changing font size"), timeout = 1.0}
    UIManager:show(info)
    self:onChangeSize("increase")
```

The above code won't work because it blocks at `self:onChangeSize("increase")`. However, if you are forcing a redraw manually, you don't need to schedule the blocking code in `nextTick` because the screen is already updated before reaching the blocking call.

If you want to use `nextTick`, then you don't need a manual force redraw because blocking code is scheduled to run asynchronously. The UI will get updated after this function returns, then uimanager will check the task queue and pick up the blocking callback.

See second tip in my comment ;)